src/libraries/System.ServiceProcess.ServiceController/src/System/ServiceProcess/ServiceBase.cs

```diff
@@ -72,6 +72,26 @@ public unsafe void RequestAdditionalTime(int milliseconds)
             }
         }
 
+#if NETCOREAPP
+        /// <summary>
+        /// When this method is called from OnStart, OnStop, OnPause or OnContinue,
+        /// the specified wait hint is passed to the
+        /// Service Control Manager to avoid having the service marked as not responding.
+        /// </summary>
+        /// <param name="time"></param>
```

Comment:
We should not have empty `<param>` tags. Eventually, we will generate docs from these. Please add something like "The requested time." and fix up RequestAdditionalTime(int milliseconds) to "The requested time in milliseconds."